scripts/libgridss.R

```diff
@@ -817,6 +817,10 @@ align_breakpoints <- function(vcf, align = c("centre"), is_higher_breakend = nam
   # Update REF base to reflect new positions
   if (!is.null(refgenome)) {
     chr = seqnames(rowRanges(vcf))
+    # Ensure chromosome names start with "chr"
```

Comment:
Let's make it more specific to hg19? 
Some of the chromosomes in hg19 do not start with "chr"